src/components/button/button.js

```diff
@@ -55,11 +61,6 @@ const btnProps = {
   }
 }
 
-const linkProps = linkPropsFactory()
-delete linkProps.href.default
-delete linkProps.to.default
-const linkPropKeys = keys(linkProps)
-
 export const props = { ...linkProps, ...btnProps }
```

Comment:
Looks like btn and link props got flipped again

Reply:
Yeah, I flipped the as they were before and you have suggested ;)

Reply:
I thought I suggest btnProps first, as the component is a button first, and link second ;-)

Reply:
![image](https://user-images.githubusercontent.com/2781561/81750352-b5350980-9483-11ea-96d3-cb40eb438f26.png)